daemon.js

```diff
@@ -250,6 +252,7 @@ export async function main(ns) {
     stockMode = (options.s || options['stock-manipulation'] || options['stock-manipulation-focus']) && !options['disable-stock-manipulation'];
     stockFocus = options['stock-manipulation-focus'] && !options['disable-stock-manipulation'];
     useHacknetNodes = options.n || options['use-hacknet-nodes'] || options['use-hacknet-servers'];
+    hacknetUsePercentage =  Math.max(options['hacknet-use-ram-percentage'])
```

Comment:
```suggestion
    hacknetUsePercentage = options['hacknet-use-ram-percentage']
```
Not sure why the max was there, it would do nothing